Tasks/plane.js

```diff
@@ -1,17 +1,15 @@
 // Make nested array plane
 
```

Comment:
Again, i suggest to add 'use strict' here. Also, the indentation in the code is uneven again - use two-space one.